device/sycl/src/clusterization/clusterization_algorithm.sycl

```diff
@@ -103,15 +104,15 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
 
                     // Barrier used in the algorithm.
                     traccc::sycl::barrier barry_r(item);
+                    device::thread_id1 thread_id(item);
```

Comment:
Eventually this should be declared `const` as well.

Reply:
True, fixed.